ctyparser/bigcty.py

```diff
@@ -109,7 +109,8 @@ def import_dat(self, dat_file: Union[str, os.PathLike]) -> None:
                     cty_dict[segments[7]] = {'entity': segments[0], 'cq': int(segments[1]),
                                              'itu': int(segments[2]), 'continent': segments[3],
                                              'lat': float(segments[4]), 'long': float(segments[5]),
-                                             'tz': -1*float(segments[6]), 'len': len(segments[7])}
+                                             'tz': -1*float(segments[6]), 'len': len(segments[7]),
+                                             'main': segments[7]}
```

Comment:
I think a better/more informative name would be `primary_pfx`

```suggestion
                                             'primary_pfx': segments[7]}
```

Reply:
good idea